subscriptions/management/commands/add_prepend_next_to_subscriptions.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Active subscription holders need to be informed via audio file "
+            "about the new missed call service.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--audio', type=str,
+            help='Audio file containing the notification of the new missed'
+            ' call service.')
+
+    def handle(self, *args, **options):
+        audio_file = options['audio']
+        if not audio_file:
+            raise CommandError('--audio_file is a required parameter')
```

Comment:
The argument is called `--audio`, I think rather change the argument to `audio-file` instead of changing this.